pkg/controller/uniteddeployment/uniteddeployment_controller_statefulset_test.go

```diff
@@ -1689,6 +1689,24 @@ func clean(g *gomega.GomegaWithT, c client.Client) {
 		return nil
 	}, timeout, time.Second).Should(gomega.Succeed())
 
+	deploymentList := &appsv1.DeploymentList{}
```

Comment:
Why do you change this file? I think these tests are for statefulset subset only.

Reply:
I thought that this `clean` helper function did not have to be duplicated separately for Deployment objects.

The function seemed generic enough to add deployment support directly inside (since it already handles both statefulsets and advancedStatefulsets).

Let me know if you would still like me to separate it out.

Reply:
Ok. Fair enough. Then please add a comment for the `clean` method saying that it is shared by all subset workload tests. Please check cloneset test as well to see if it can use it.

Reply:
Seems like cloneset tests are already referencing this `clean` method.